airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -1,23 +1,56 @@
+from airflow.exceptions import AirflowException
 from airflow.sensors.sql_sensor import SqlSensor
 from airflow_clickhouse_plugin.hooks import clickhouse_hook
 
 
 class ClickHouseSqlSensor(SqlSensor):
     def __init__(
         self,
-        clickhouse_conn_id: str = "clickhouse_default",
-        database=None,
+        clickhouse_conn_id: str = 'clickhouse_default',
+        database: str = None,
         *args,
-        **kwargs
+        **kwargs,
     ):
         super(ClickHouseSqlSensor, self).__init__(conn_id=None, *args, **kwargs)
         self._conn_id = clickhouse_conn_id
         self._database = database
 
-    def _get_hook(self):
-        return clickhouse_hook.ClickHouseHook(
+    def poke(self, context):
+        hook = clickhouse_hook.ClickHouseHook(
```

Comment:
This looks terrible though I don't see a better way to implement this except of monkey-patching `Connection.get_hook`. Please add a comment with a permalink to Airflow code which is copy-pasted here.